src/libraries/System.Windows.Extensions/src/System/Media/SoundPlayer.cs

```diff
@@ -261,16 +262,7 @@ private void LoadSync()
             // setup the http stream
             if (_uri != null && !_uri.IsFile && _stream == null)
             {
-#pragma warning disable SYSLIB0014 // WebRequest, HttpWebRequest, ServicePoint, and WebClient are obsolete. Use HttpClient instead.
-                WebRequest webRequest = WebRequest.Create(_uri);
-#pragma warning restore SYSLIB0014
-                webRequest.Timeout = LoadTimeout;
-
-                WebResponse webResponse;
-                webResponse = webRequest.GetResponse();
-
-                // now get the stream
-                _stream = webResponse.GetResponseStream();
+                _stream = _httpClient.GetStreamAsync(_uri).Result;
```

Comment:
```suggestion
                _stream = _httpClient.GetStreamAsync(_uri).GetAwaiter().GetResult();
```

Since this function is synchronous, I'm wondering if we should use `HttpClient.Send` instead to avoid sync-over-async
Any thought on that @manickap @stephentoub ?

Reply:
`HttpWebRequest` is using the sync APIs already so we should likely do so here as well, or this change would actually regress perf.